.nvmrc

```diff
@@ -1 +1 @@
-14.4.0
+12.14
```

Comment:
This version conflicts with the version in the github [workflow file](https://github.com/sbrudz/accelerate-metrics/pull/4/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R12) - is that intended?

Reply:
That was an oversight.  Good catch!

Reply:
Fixed